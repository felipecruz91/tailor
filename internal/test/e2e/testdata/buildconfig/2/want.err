Diff not performed due to misconfiguration
