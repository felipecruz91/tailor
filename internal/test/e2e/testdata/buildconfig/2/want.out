Comparing templates in buildconfig/2 with OCP namespace {{ .Project }}.
Limiting resources to bc.
Found 1 resource in OCP cluster (current state) and 0 resources in processed templates (desired state).

No items where found in desired state. Possible reasons are:
* No templates are located in buildconfig/2
* No templates contain resources of kinds: bc

Refusing to continue without --force
